Makefile

```diff
@@ -97,14 +102,12 @@ images:
 		--@io_bazel_rules_go//go/config:pure \
 		//build:server-images
 
-images_push: 
+.PHONY: images_push
+images_push:
 	APP_VERSION=$(APP_VERSION) \
 	DOCKER_REGISTRY=$(DOCKER_REGISTRY) \
 	bazel run \
 		--stamp \
 		--platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 \
 		--@io_bazel_rules_go//go/config:pure \
 		//:images.push
-
-ctl:
-	bazel build //cmd/ctl
```

Comment:
note for reviewers: because `ctl` is under the `cmd` directory, this target is also provided by the `$(CMDS)` target above, meaning this is a duplicate which was causing warnings every time the Makefile was used.

Reply:
Thank you for that, I always thought "I shall fix this warning" and never did